packages/cli/api-importers/openapi/openapi-v2-importer/src/3.1/paths/operations/WebhookConverter.ts

```diff
@@ -68,28 +68,21 @@ export class WebhookConverter extends AbstractOperationConverter {
             group,
             method
         });
-        if (requestBody == null || !("contentType" in requestBody) || !requestBody.contentType?.includes("json")) {
-            errorCollector.collect({
-                message: "Skipping webhook because non-json request body",
-                path: this.breadcrumbs
-            });
-            return undefined;
-        }
```

Comment:
not sure we need this, hypothetically we may want to document any sort of request payload in the webhook request body